blog-contribution-guidelines

```diff
@@ -0,0 +1,78 @@
+## How to get started contributing to the Falco blog
+
+Welcome to the Falco community blog. This blog serves as an educational resource for community members, maintainers, and contributors. 
+
+The process is meant to be self-serve but if you experience any difficulties reach out to us at contact us via the [Kubernetes Slack on the #falco](https://kubernetes.slack.com/messages/falco) channel. 
+
+**TLDR;** submit your [blog PR](https://github.com/falcosecurity/falco-website/pulls) here. 
```

Comment:
```suggestion
**TLDR;** submit your blog PR [here](https://github.com/falcosecurity/falco-website/pulls).
```
I would set the link on the `here` word, feel free to change it if you don't like it :)